src/esm_master/task.py

```diff
@@ -534,3 +550,181 @@ def output_steps(self):
             print("    Executing commands in this order:")
             for command in self.shown_command_list:
                 print("        ", command)
+
+    # Repository methods
+    # ------------------
+    def num_of_get_commands(self):
+        """
+        Defines ``self.num_get_commands`` which accounts for the total number of get
+        commands for the current ``esm_master`` operation
+        """
+        self.num_get_commands = 0
+        for command in self.command_list:
+            for subtask in self.subtasks:
+                if command in subtask.package.command_list.get("get", []):
+                    self.num_get_commands += 1
+                    break
```

Comment:
Should this thing also return something?
```suggestion
    def num_of_get_commands(self):
        """
        Defines ``self.num_get_commands`` which accounts for the total number of get
        commands for the current ``esm_master`` operation
        """
        self.num_get_commands = 0
        for command in self.command_list:
            for subtask in self.subtasks:
                if command in subtask.package.command_list.get("get", []):
                    self.num_get_commands += 1
                    break
        return self.num_get_commands # (Not strictly needed, but might be nice?)
```

Reply:
No need to return anything, it's all contained in attributes of the object